UnityProject/Assets/Scripts/Items/Tool/Multitool/Multitool.cs

```diff
@@ -92,21 +92,32 @@ public void PrintElectricalThings(PositionalHandApply interaction)
 			var matrix = interaction.Performer.GetComponentInParent<Matrix>();
 			var electricalNodes = matrix.GetElectricalConnections(localPosInt);
 
+			APCPoweredDevice device = default;
+			bool deviceFound = interaction.TargetObject != null && interaction.TargetObject.TryGetComponent(out device);
+
 			string message = "The multitool couldn't find anything electrical here.";
-			if (electricalNodes.Count > 0)
+			if (deviceFound || electricalNodes.Count > 0)
 			{
-				message = "The multitool's display lights up:\n";
-			}
+				StringBuilder sb = new StringBuilder("The multitool's display lights up.\n</i>");
 
-			StringBuilder sb = new StringBuilder(message);
-			foreach (var node in electricalNodes)
-			{
-				sb.AppendLine(node.ShowInGameDetails());
+				if (deviceFound)
+				{
+					sb.AppendLine(device.RelatedAPC == null
+							? $"{device.gameObject.ExpensiveName()}</b> is not connected to an APC!"
+							: $"<b>{device.gameObject.ExpensiveName()}</b>: {device.Wattusage.ToEngineering("W")} " +
+									$"({device.RelatedAPC.Voltage.ToEngineering("V")})");
+				}
+				foreach (var node in electricalNodes)
+				{
+					sb.AppendLine(node.ShowInGameDetails());
+				}
+
+				message = sb.ToString() + "<i>";
```

Comment:
Why not append the closure of the message and only use ToString when displaying the text?